<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addstudent</class>
 <widget class="QDialog" name="addstudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>841</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Niagara Engraved</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#title{

font-size:50px;
border: 1px solid black;
border-radius: 10% ;

}</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>501</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
font-size:20px;
sont-style:bolder;
}</string>
   </property>
   <property name="title">
    <string>Student Information:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">#label{
font-size:15px;
}</string>
         </property>
         <property name="text">
          <string>First Name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">#label_2{
font-size:15px;
}</string>
         </property>
         <property name="text">
          <string>Last Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">#label_3{
font-size:15px;
}</string>
         </property>
         <property name="text">
          <string>Adresse:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">#label_4{
font-size:15px;
}</string>
         </property>
         <property name="text">
          <string>Class :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="styleSheet">
          <string notr="true">#comboBox{
broder-radius:10%;
}</string>
         </property>
         <item>
          <property name="text">
           <string>1er annee</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2eme</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3eme</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4eme</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5eme</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">#label_5{
font-size:15px;
}</string>
         </property>
         <property name="text">
          <string>password:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="inputMethodHints">
          <set>Qt::ImhHiddenText</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>411</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="submit">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
